Fix Rubocop, nvm and webpacker setup

novotnyjakub opened the pull request:
### Context

This ensures the development can start without existing Rubocop issues and deprecation warnings.

### Changes proposed in this pull request

* Rubocop config properly inherits from gov.uk and the existing code complies (few tiny changes)
* the .nvmrc file version is inline with the yarn requirement
* webpacker config doesn't use deprecated fields

### Guidance to review

Nothing specific here.

novotnyjakub committed:
Match nvmrc version with yarn

novotnyjakub committed:
Fix Rubocop

This involves making sure the rubocop-govuk gem is properly configured.

Also I removed the duplicated lint_ruby and lint_scss rake tasks, they
were just renaming the existing ones. The lint tasks don't seem to be
working for the moment, I suspect it may be Ruby 2.7 related. But
running `bundle exec rubocop` works well.

novotnyjakub committed:
Fix deprecation error in webpacker.yml

goodviber reviewed (changes requested):
The spec_helper file is double commented?

novotnyjakub commented:
Yeah, it's a sample piece of config, which has comments and the whole piece was commented out by a block comment, which doesn't comply with Rubocop rules. The only option would be to remove the sample altogether, but I don't mind this, so it's up to you.

goodviber commented:
I would normally uncomment line 60 config.example_status_persistence_file_path = "spec/examples.txt so we can use --only failures and put spec/examples.txt in gitignore, some of the other configs might be useful also, we can just comment out the individual lines we dont need presently. Gov Rubocop is a complete pain...

novotnyjakub committed:
Improve RSpec config and remove double comments

As per PR comments, I uncommented the additional RSpec config and
allowed failed examples to be stored in a git ignored text file.
